packages/fleet-agent/package.yaml

```diff
@@ -1,2 +1 @@
 url: https://github.com/rancher/fleet/releases/download/v0.3.5/fleet-agent-0.3.5.tgz
```

Comment:
@cbron should we handle adding `version` in this PR itself based on the prospective versions in our versioning doc? Developers can always update it after the fact.

Reply:
we are doing this